bin/shredder

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -ex
+
+find $@ -type f -exec shred --remove {} +
```

Comment:
Hmm. I thought I didn't want this because multiple args would end up getting treated as one, but it looks like I was wrong about that.